static/styles/zulip.css

```diff
@@ -3143,6 +3143,103 @@ select.inline_select_topic_edit {
     font-weight: 600;
 }
 
+.reportmsg-box {
+    padding: 10px 15px 10px 10px;
+    display: flex;
+    margin-top: 5px;
+    margin-bottom: 10px;
+    background-color: hsl(202, 56%, 91%);
+    border: 1px solid hsla(192, 19%, 75%, 0.2);
+
+    .radio-input-parent {
+        float: left;
+        padding-right: 20px;
+        padding-bottom: 10px;
+
+        label.radio {
+            padding-left: 25px;
+        }
+    }
+
+    .reportmsg-details {
+        float: left;
+        flex-grow: 1;
+        margin-right: 10px;
+
+        .reportmsg-explain {
+            width: 100%;
+            height: 100px;
+        }
+    }
+
+    .reportmsg-buttons {
+        float: right;
+        margin-right: -14px;
+
+        .btn-wrapper {
+            .button {
+                padding: 6px 10px;
+                margin: 0;
+
+                font-size: 0.8rem;
+                font-weight: 400;
+                line-height: normal;
+                text-align: center;
+
+                background-color: hsl(0, 0%, 100%);
+                color: inherit;
+                outline: none;
+                border: 1px solid hsl(0, 0%, 80%);
+                border-radius: 2px;
+
+                box-shadow: none;
+
+                cursor: pointer;
+                transition: all 0.2s ease;
+
+                &.rounded {
+                    border-radius: 4px;
+                }
+
+                &.green {
+                    background-color: hsl(150, 31%, 53%);
+                }
+
+                &.grey {
+                    background-color: hsl(0, 0%, 67%);
+                }
+
+                &:hover,
+                &:focus {
+                    border-color: hsl(0, 0%, 60%);
+                }
+
+                &:active {
+                    border-color: hsl(0, 0%, 60%);
+                    color: inherit;
+                    background-color: hsl(0, 0%, 95%);
+                }
+
+                &.sea-green {
+                    color: hsl(156, 41%, 40%);
+                    border-color: hsl(156, 28%, 70%);
+
+                    &:hover,
+                    &:focus {
+                        border-color: hsl(156, 30%, 50%);
+                    }
+
+                    &:active {
+                        border-color: hsl(156, 30%, 40%);
+                        color: hsl(156, 44%, 43%);
+                        background-color: hsl(154, 33%, 96%);
+                    }
```

Comment:
This is a lot of button styles! I'm guessing that the actual problem that required you to copy these is that the `new-style` CSS class isn't set on an outer container.

There's a refactor we should try to do after this lands to make `new-style` the default -- last I recall, there's no major blockers for that, but deserves a #frontend discussion.

Reply:
DONE - added a TODO... I totally agree re style complexity !!!  I'm also thinking this could wait until the new UI redesign lands as well... thx!